ui/partials/perf/comparetable.html

```diff
@@ -43,7 +43,28 @@
           {{compareResult.deltaPercentage|displayPrecision}}%
         </span>
       </td>
-      <td><div ng-class="getCompareClasses(compareResult, 'bar')" style="margin-{{compareResult.marginDirection}}: {{compareResult.barGraphMargin}}%;"></div></td>
+      <td ng-if="compareResult.delta">
+        <div ng-if="!compareResult.newIsBetter" style="margin: auto; width: 80%;"
+             tooltip="Relative magnitude of change (scale from 0 - 20%+)">
+          <div class="bar bar-scale"
+               style="width: {{100 - compareResult.magnitude}}%; height: 1em; float: left;">
+          </div>
+          <div class="bar"
+               ng-class="{'bar-regression': compareResult.isRegression}"
+               style="width: {{compareResult.magnitude}}%; float: left;">
+          </div>
+        </div>
+        <div ng-if="compareResult.newIsBetter" style="margin: auto; width: 80%;"
+             tooltip="Relative magnitude of change (scale from 0 - 20%+)">
+          <div class="bar"
+               ng-class="{'bar-improvement': compareResult.isImprovement}"
+               style="width: {{compareResult.magnitude}}%; float: left;">
+          </div>
+          <div class="bar bar-scale"
+               style="width: {{100 - compareResult.magnitude}}%; float: left; ">
+          </div>
+        </div>
+      </td>
```

Comment:
This block of code makes sense and it easy to read.  My reason for a comment is that if we come back to this in the future, it might be confusing or error prone upon modifications.  I am fine with a followup to refactor this if we want to land this faster.